base/uk.ac.stfc.isis.ibex.configserver/src/uk/ac/stfc/isis/ibex/configserver/Configurations.java

```diff
@@ -76,6 +78,7 @@ public Configurations() {
 		editing = registerForClose(new ConfigEditing(server));
 
 		iocControl = new IocControl(server);
+		moxaMappings = new MoxaMappings(server);
```

Comment:
Could this not just be:
```suggestion
		moxaMappings = new UpdatedObservableAdapter<>(server.moxaMappings());
```

Then we just need to modify the line adding a listener in `MoxasViewModel` and we can get rid of the whole `MoxaMappings` class which looks a bit like unncessary boilerplate (unless I missed something!)

Reply:
it is unnecessary boilerplate I think, though I'm struggling figuring out how to tie the viewmodel in. when constructing it i'm not sure what to pass - just the config server itself i guess?

Reply:
think i've figured it out. just tidying up a bit then will push changes